reports/historical.py

```diff
@@ -231,13 +235,13 @@ def import_from_file(self, fname, coverage, region=None, country=None):
         elif country:
             self.log.info("Importing for country %s" % country)
             key = canon(country)
-
         for old_sheet, new_sheet in self.historical_sheets(coverage):
             # find matching sheet name
             ws = None
             for name in wb.sheetnames:
-                if name == old_sheet or name.startswith(old_sheet + ' '):
+                if name == old_sheet[0] or name.startswith(old_sheet + ' '):
```

Comment:
Again, what's happening here?

Reply:
`old_sheet` returns `1F` (as an example for the first sheet) and `new_sheet` returns `1`. That's why I picked the first letter. Going forward, I'll be making use of a mapping, which maps `old_sheet` keys to `new_sheet` keys. That way, we won't have this code here. @kilemensi

Reply:
Not sure I fully get you yet:

Aren't we supposed to change:

https://github.com/CodeForAfrica/gmmp/pull/115/files#diff-e0b8644d16d155669d4a77d2a2863f07ae2ce51bbacfbf107d4d11bfb3f91773R1219-R1220

to: 

```json
"1": {
```
?

Reply:
@kilemensi I'm not sure I get this...

Reply:
What I mean @ascii-dev is lets change where the info is coming from: since you were changing oldsheet, I assumed you wanted to change where we're reading and hence me point it above.

So may be I should ask: why do you need to get 1 from 1F?

Reply:
`1F` refers to a specific worksheet in `2010`. This same worksheet was renamed to `1` in `2015`. We need a way to be able to deduce that the `2015` data for `1F` can be found in `1` of the `new_sheet`. The current solution I have up is just a hack to get that running. Currently working on making the codebase better but the idea behind getting `1` from `1F` still remains the same. @kilemensi

Reply:
Just to add to what @kilemensi said, what I think we should be changing is https://github.com/CodeForAfrica/gmmp/blob/master/reports/report_details.py#L67
since the new historical is the 2015 report, its name becomes the new historical and we get rid of the 2010 historical worksheet.
Something like 
```
'ws_01': {
        'name': '1',
        'title': 'Participating Countries',
        'desc': 'Breakdown of all media by region',
        'reports': ['global'],
        'historical': '1',
    }
```

Reply:
I'm trying to make changes that will still preserve the `2010` historical functionalities, because it's possible we might need it in the future... @esirK

Reply:
@ascii-dev I'm not sure about the 
> 2010 historical functionalities,

part but as far as data is concerned, I don't think we need to do anything in order to have 2010 data. If you look at the `json` file, https://github.com/CodeForAfrica/gmmp/blob/master/historical.json#L3
we ready have all the data from `1995-2010` so when we import 2015 report it will be added to this and the keys updated appropriately.